sympy/printing/fcode.py

```diff
@@ -178,6 +172,32 @@ def _print_Piecewise(self, expr):
                                       "inline operators is not supported in "
                                       "standards earlier than Fortran95.")
 
+    def _print_Assignment(self, expr):
```

Comment:
It would be clearer if the the content of `_print_Assignment` was into `_print_sign`.
What prevents you to use only one method?